internal/engine/experiment/tor/tor.go

```diff
@@ -330,75 +325,59 @@ func maybeScrubbingLogger(input model.Logger, kt keytarget) model.Logger {
 	return &scrubber.Logger{Logger: input}
 }
 
-func (rc *resultsCollector) defaultFlexibleConnect(
-	ctx context.Context, kt keytarget,
-) (tk oonitemplates.Results, err error) {
-	logger := maybeScrubbingLogger(rc.sess.Logger(), kt)
+// defaultFlexibleConnect is the default implementation of the
+// rc.flexibleConnect testable function.
+//
+// Arguments:
+//
+// - ctx is the context for timeout/cancellation;
+//
+// - kt contains information about the target.
+//
+// Returns:
+//
+// - tk is the measurement, which is always non nil because
+// the measurex "easy" API provides this guarantee;
+//
+// - failure is nil or an OONI failure string.
+func (rc *resultsCollector) defaultFlexibleConnect(ctx context.Context,
+	kt keytarget) (tk *measurex.ArchivalMeasurement, failure *string) {
+	mx := measurex.NewMeasurerWithDefaultSettings()
+	mx.Begin = rc.measurement.MeasurementStartTimeSaved
+	mx.Logger = maybeScrubbingLogger(rc.sess.Logger(), kt)
```

Comment:
The logger configured into the `mx` is a `maybeScrubbingLogger` like it was before.